libs/wingsdk/src/std/map.ts

```diff
@@ -72,29 +82,37 @@ export class ImmutableMap {
 export class MutableMap extends ImmutableMap {
   /**
    * Removes all elements
+   * @macro $self$={}
```

Comment:
does this work? Do we have a test for this?

Reply:
Great catch, I thought I saw a test for this but my eyes must have mixed up `copy` and `clear` This in-fact did NOT work because you cant reassign const 😅 I updated it to iterate the keys and call delete. Also added test to confirm.